salt/apache/files/sites/coalition.conf.include

```diff
@@ -0,0 +1,23 @@
+DocumentRoot /home/coalition/public_html
+<Directory /home/coalition/public_html>
+    Options FollowSymLinks
+    AllowOverride Limit Options FileInfo
+    DirectoryIndex index.php
+    Require all granted
+</Directory>
+
+<Directory /home/coalition/public_html/wp-content>
+    Options FollowSymLinks
+    Require all granted
+</Directory>
+
+<FilesMatch \.php$>
+  SetHandler "proxy:unix:{{ php_socket }}|fcgi://localhost/"
+</FilesMatch>
+
+<FilesMatch "^(wp-admin|wp-login\.php)">
+    AuthType Basic
+    AuthName "Restricted Area"
+    AuthUserFile /etc/apache2/.htpasswd-coalition
+    Require valid-user
+</FilesMatch>
```

Comment:
WordPress' authentication is sufficient for security assuming we have good passwords.
I added basicauth to specifically prevent bots and undirected "drive-by" attacks which are common because of how popular WP is.
Blocking attacks at this level also stops the bots before they reach PHP mitigating any effect on other site visitors.

Reply:
Sounds good